### PR TITLE
Replace Font Awesome with Fork Awesome

### PR DESCRIPTION
**Pull request by devnix**

### CODE REVIEW

**Review by Fraizeraust — APPROVED** [review]
The changes look good to me.
Capture

**Comment by Fraizeraust** [comment]
As a word of advice, the title of the PR should be "Replace Font Awesome with Fork Awesome", reflecting the changes provided by the pull request. Also we don't use the ticket IDs as labels within commit titles.

File: themes/reactos/layouts/partials/shared-head.html [hunk 1]
```diff
@@ -2,7 +2,8 @@
 	<link rel="stylesheet" href="{{ "css/font-roboto.css" | relURL}}" type='text/css'>
 
 	<!-- Bootstrap and Font Awesome css -->
```
Comment (learn-more) [inline themes/reactos/layouts/partials/shared-head.html:4]: Nit: it's not font awesome anymore

**Review by learn-more — APPROVED** [review]
LGTM.
2 nitpicks: themes/reactos/README.md still mentions font awesome, would be nice to have that updated to include fork awesome (and the specific version used),
and the other is in shared-head.html

**Comment by learn-more** [comment]
Thanks!

**Pull request merged**